modules/nuxt-basic-auth.yml

```
@@ -0,0 +1,16 @@
+name: nuxt-basic-auth
+description: Nuxt 3 Module for Basic Authentication
+repo: kgierke/nuxt-basic-auth
+npm: '@kgierke/nuxt-basic-auth'
+icon: ''
+github: https://github.com/kgierke/nuxt-basic-auth
+website: https://github.com/kgierke/nuxt-basic-auth
+learn_more: ''
+category: Devtools
```

Is this really Devtools? Maybe

```suggestion
category: Security
```